cmd/oras/root/backup.go

```diff
@@ -163,6 +163,16 @@ func runBackup(cmd *cobra.Command, opts *backupOptions) error {
 	case outputFormatDir:
 		dstRoot = opts.output
 	case outputFormatTar:
+		// test if the output file can be created and fail early if there is an issue
+		fp, err := os.Create(opts.output)
```

Comment:
Is it safter to `os.OpenFile` without `O_TRUNC`?

Reply:
updated